community/kernel/src/main/java/org/neo4j/kernel/impl/api/KernelTransactionImplementation.java

```diff
@@ -793,20 +805,20 @@ public void visitDegree( int type, int outgoing, int incoming )
                         for ( Integer label : added )
                         {
                             // untyped
-                            counts.incrementRelationshipCount( label, -1, -1, outgoing );
-                            counts.incrementRelationshipCount( -1, -1, label, incoming );
+                            counts.incrementRelationshipCount( label, ANY_RELATIONSHIP_TYPE, ANY_LABEL, outgoing );
+                            counts.incrementRelationshipCount( ANY_LABEL, ANY_RELATIONSHIP_TYPE, label, incoming );
                             // typed
-                            counts.incrementRelationshipCount( label, type, -1, outgoing );
-                            counts.incrementRelationshipCount( -1, type, label, incoming );
+                            counts.incrementRelationshipCount( label, type, ANY_LABEL, outgoing );
+                            counts.incrementRelationshipCount( ANY_LABEL, type, label, incoming );
                         }
                         for ( Integer label : removed )
                         {
                             // untyped
-                            counts.incrementRelationshipCount( label, -1, -1, -outgoing );
-                            counts.incrementRelationshipCount( -1, -1, label, -incoming );
+                            counts.incrementRelationshipCount( label, ANY_RELATIONSHIP_TYPE, ANY_LABEL, -outgoing );
+                            counts.incrementRelationshipCount( ANY_LABEL, ANY_RELATIONSHIP_TYPE, label, -incoming );
                             // typed
-                            counts.incrementRelationshipCount( label, type, -1, -outgoing );
-                            counts.incrementRelationshipCount( -1, type, label, -incoming );
+                            counts.incrementRelationshipCount( label, type, ANY_LABEL, -outgoing );
+                            counts.incrementRelationshipCount( ANY_LABEL, type, label, -incoming );
```

Comment:
This code snippet is used in several places, perhaps there is value in moving it into a method.